src/main/java/frc/robot/commands/mailbox/FireNoteRoutineNoLimitSwitch.java

```diff
@@ -17,7 +17,11 @@
 /**
  * Scores the note into the amp. Raises the mailbox, runs the mailbox belts, and then runs the index
  * belts to send the note into the belts without a limit switch.
+ *
+ * @deprecated This class does not work consistantly when there is an intake limit switch present,
+ *     which will be present for the forseeable future.
```

Comment:
https://duckduckgo.com/?q=spell+consistantly and https://duckduckgo.com/?q=spell+forseeable